util/__init__.py

```diff
@@ -0,0 +1,4 @@
+import re

```

Comment:
This folder/file needs to be moved to the [getnotes](https://github.com/Crossedfall/crossed-cogs/tree/ss13/master/getnotes) directory